public/js/app.js

```diff
@@ -173,8 +174,28 @@ jitterbug = {
     });
   },
 
+  toggleInactive: function() {
+    var inactiveCheckboxes = $('.inactive input:checkbox');
+    inactiveCheckboxes.click(function(event) {
+      var makeInactive = $(this).is(':checked');
+      var route = makeInactive == true ? '/users/inactivate'
```

Comment:
Are you able to get rid of the `== true` and `=== true` below since makeInactive should just evaluate to true or false?

Reply:
yes! i did this in the other toggle functions as well.